### PR TITLE
Make DG more consistent

### PR DESCRIPTION
**Pull request by oliviajohansen**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ title: Developer Guide
     - [3.3. List Features](#33-list-features)
         * [3.3.1 Implementation](#331-implementation)
         * [3.3.2 Design Consideration - **List Recipes**](#332-design-consideration-list-recipe)
-            * [Aspect: Concern while adding a new feature](#3321-aspect)
+            * [Aspect 1 : Concern while adding a new feature](#3321-aspect-1)
+            * [Aspect 2: What and how to show recipe information](#3322-aspect-2)
```
Comment (caitlinjee) [inline docs/DeveloperGuide.md]: what do you mean what and how?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -369,10 +372,12 @@ Command and Parser makes use of Substitutability:
 * `EatRecipeCommand` extends `Command`
 * `EatRecipeCommandParser` implements `Parser<EatRecipeCommand>` <br><br>
 
-The following activity diagram shows how eat recipe operation works:
+The following two diagrams shows how eat recipe operation works when `execute(eatR 1)` is called:
```
Comment (caitlinjee) [inline docs/DeveloperGuide.md]: this one should add the "" so its execute("eatr 1")?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -496,7 +520,8 @@ remove(key) | `removeRecipe(key)` | `removeIngredient(key)` | `removeConsumption
 </div>
 
 Given below is an example usage scenario and how the mechanism behaves:
-1. User inputs delete command with the item index, to delete that specific item from the respective list.
+1. User inputs delete command with the item index, to delete that specific item from their respective recipe
+ list, fridge or consumption list.
```
Comment (caitlinjee) [inline docs/DeveloperGuide.md]: i think i changed fridge to ingredient list to match the ug

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -810,12 +839,17 @@ Clear features - `Clear Recipes`, `Clear Ingredients` and `Clear Consumption` al
 Command and Parser make use of Substitutability:
 * `ClearRecipeCommand`, `ClearIngredientCommand` and `ClearConsumptionCommand` extends `Command` <br><br>
 
-The following activity diagram shows how clear operation generally works:
-![ClearActivityDiagram](images/implementation/activityDiagrams/ClearCommandActivityDiagram.png)
-
-The following sequence diagram shows how clear operation works when `execute(clearR)`, `execute(clearF)` or
+The following two diagrams shows how clear operation generally works when clear recipes: <br> 
+`execute(clearR)` <br>
```
Comment (caitlinjee) [inline docs/DeveloperGuide.md]: i think also missed out the " " around the commands

File: docs/team/oliviajohansen.md [hunk 5]
```diff
@@ -52,7 +52,9 @@ Given below are my contributions to the project.
 
 * **Enhancements to existing features**:
   * Added support for quantity for ingredients (Pull requests [\#82](https://github.com/AY2021S1-CS2103T-W10-2/tp/pull/82))
-    * Justification: This feature allow users to better keep track of quantity of ingredients in the fridge.
+    * Justification: This feature allow users to better keep track of quantity of ingredients in the fridge. The
+     flexible format of quantity (fraction, decimal or integers followed by units or just a string) allowed
```
Comment (caitlinjee) [inline docs/team/oliviajohansen.md]: allows users?

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -535,20 +560,18 @@ Command and Parser make use of Substitutability:
 * `EditRecipeCommandParser` implements `Parser<EditRecipeCommand>`
 * `EditIngredientCommandParser` implements `Parser<EditIngredientCommand>` <br><br>
 
-The following activity diagram shows how edit operation generally works when a recipe is edited: <br>
+The following two activity diagrams shows how edit operation generally works when a recipe is edited: <br>
```
Comment (TianYong-Goh) [inline docs/DeveloperGuide.md]: I thought only have one activity diagram?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (oliviajohansen) [reply docs/DeveloperGuide.md]: lol what fields of recipe to display and how to display some fields that could be very long

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (oliviajohansen) [reply docs/DeveloperGuide.md]: ok

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (oliviajohansen) [reply docs/DeveloperGuide.md]: ok

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (oliviajohansen) [reply docs/DeveloperGuide.md]: ok

File: docs/team/oliviajohansen.md (hunk 5 above)
Comment (oliviajohansen) [reply docs/team/oliviajohansen.md]: whats wrong?

File: docs/DeveloperGuide.md (hunk 6 above)
Comment (oliviajohansen) [reply docs/DeveloperGuide.md]: oh i meant to say following two diagrams

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (caitlinjee) [reply docs/DeveloperGuide.md]: ooh maybe can word beter? like what info to show and how to show them?

File: docs/team/oliviajohansen.md (hunk 5 above)
Comment (caitlinjee) [reply docs/team/oliviajohansen.md]: ohoh the 2nd allow, the flexible format... just a string) allowed -> allows to match 1st allow?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (oliviajohansen) [reply docs/DeveloperGuide.md]: okok

File: docs/team/oliviajohansen.md (hunk 5 above)
Comment (oliviajohansen) [reply docs/team/oliviajohansen.md]: oo yes

**Pull request merged**